### DISCUSSION TITLE
2 axis Laser Cut

### DISCUSSION THREAD

**Discussion by mimarkun** [fellesverkstedet/Bark-beetle-parametric-toolpaths#87] [Q&A]
Hi, guys. Thanks for sharing such an interesting code. I’ve downloaded BarkBeetle and am trying to figure out how to use it. If be honest it is not very easy. Have a question, is it possible to generate G-code for a 2axis laser cut? thanks

**Comment by JensDyvik** [comment]
Hi. How is it going with your laser cutting project? 2 axis laser cutting is not directly supported in the current version of bark beetle, but should be possible. What kind of firmware is your laser cutter running?

**Reply by mimarkun** [reply]
Hi, thanks for your reply. I did everything from scratch in the GH Python and now it works as I need it.

**Comment by JensDyvik** [comment]
That's super. GH python is nice and powerful :)
